dev/provision_salt.sh

```diff
@@ -7,25 +7,34 @@
 set -e
 
 MINION_PLIST_PATH=/Library/LaunchDaemons/com.saltstack.salt.minion.plist
+LINUX_SALT_CLIENT_PATH="$HOME/salt-client"
+SALT_VERSION='3006.9'
 
 # Installs salt minion.
 # Pins the version to 2019.2.0 and Python 2 to be compatible with Fuchsia salt master.
 function install_salt() {
-  if [[ "$(uname)" == 'Darwin' ]]; then
-    curl https://repo.saltproject.io/osx/salt-3002.9-py3-x86_64.pkg -o /tmp/salt.pkg
+  OS="$(uname)"
+  if [[ "$OS" == 'Darwin' ]]; then
+    curl "https://packages.broadcom.com/artifactory/saltproject-generic/macos/$SALT_VERSION/salt-3006.9-py3-x86_64.pkg" -o /tmp/salt.pkg
```

Comment:
Should we use `${SALT_VERSION}` for the other hardcoded version names here and below?

Reply:
good catch

Reply:
Done